ham/utils/constants.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+# PATH constants
+HAM_PATH = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+SYMLINK_NAME = os.path.join(HAM_PATH, '..', 'current')
```

Comment:
I guess that placing the current symlink here will not behave well when using an installation from pip